proprietary/tokens/src/components/ams/table-of-contents.tokens.json

```
@@ -0,0 +1,56 @@
+{
+  "ams": {
+    "table-of-contents": {
+      "color": { "value": "{ams.color.primary-black}" },
+      "font-family": { "value": "{ams.text.font-family}" },
+      "font-size": { "value": "{ams.text.level.5.font-size}" },
+      "font-weight": { "value": "{ams.text.font-weight.normal}" },
+      "gap": { "value": "{ams.space.inside.lg}" },
```

The gap is 16 pixels in Figma.

```suggestion
      "gap": { "value": "{ams.space.inside.md}" },
```

---

In the amsterdam.nl figma, guess i've taken the value of the design system figma.